enos/modules/ec2_info/main.tf

```diff
@@ -1,28 +1,53 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: BUSL-1.1
 
+# Note: in order to use the openSUSE Leap AMIs, the AWS account in use must "subscribe"
+# and accept SUSE's terms of use. You can do this at the links below. If the AWS account
+# you are using is already subscribed, this confirmation will be displayed on each page.
+# openSUSE Leap arm64 subscription: https://aws.amazon.com/marketplace/server/procurement?productId=a516e959-df54-4035-bb1a-63599b7a6df9
+# openSUSE leap amd64 subscription: https://aws.amazon.com/marketplace/server/procurement?productId=5535c495-72d4-4355-b169-54ffa874f849
```

Comment:
Did we already subscribe to these in the vault_enterprise_ci account as well? I guess we'll find out with the ent PR

Reply:
Not yet but I will do that with the ent PR.

Reply:
I feel like we need to put this information in the scenario descriptions so it's easy to see. I'll make a note do that in my next PR that adds descriptions when I rebase on this.